cl/search/management/commands/generate_opinion_embeddings.py

```diff
@@ -0,0 +1,84 @@
+from cl.lib.command_utils import VerboseCommand
+from cl.search.models import Opinion
+from cl.search.tasks import create_opinion_text_embeddings
+
+
+def send_batch(
+    batch: list[int],
+    embedding_queue: str,
+    upload_queue: str,
+    database: str,
+    bucket_name: str,
+) -> None:
+    create_opinion_text_embeddings.si(
+        batch,
+        upload_queue,
+        database,
+        bucket_name,
+    ).apply_async(queue=embedding_queue)
```

Comment:
At least consider doing a celery chain here. If you do that, you can pass fewer arguments to the task, and I think your log gets simpler. Lots of examples in our code. Look for `chain(`.